src/deprecated.jl

```diff
@@ -0,0 +1,35 @@
+@deprecate generate_s(args...) gen(DBNVar(:sp), args...)
```

Comment:
Shouldn't this be `DBNOut(:sp)` for consistency? (also below)

Reply:
Is it because there is a single argument, rather than multiple (as below, which seem to have `DBNOut`)

Reply:
Yeah, sorry I didn't fix these correctly after getting rid of `Return`